queue/queue.py

```diff
@@ -1,15 +1,23 @@
+
+from linked_list_py import LinkedList
+
 class Queue:
   def __init__(self):
     self.size = 0
-    # what data structure should we
-    # use to store queue elements?
-    self.storage = 
+    self.storage = LinkedList()
 
   def enqueue(self, item):
-    pass
+    self.storage.add_to_tail(item)
   
   def dequeue(self):
-    pass
+    return self.storage.remove_head()
 
   def len(self):
-    pass
+    currNode = self.storage.head
```

Comment:
What's a better way to do this? Take not of the `self.size` property on the Queue class, what can you do in your `enqueue` and `dequeue` methods to keep track of the size without needing a loop here?